express-post-json/index.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const app = express();
+
+let nextId = 1;
+let grades = {};
+const arr = [];
+
+const jsonMethod = express.json();
+app.use(jsonMethod);
+
+app.post('/api/grades', (req, res) => {
+  grades = req.body;
+  req.body.id = nextId;
+  arr.push(grades);
+  res.status(201).json(grades);
+  nextId++;
+});
```

Comment:
The response should only be sent after everything else is updated, otherwise the client may receive a success response when part of the operation failed.